### PR TITLE
Playmode scenarios documentation

### PR DESCRIPTION
**Pull request by Vic-Cooper**
**Select the type of change:**
- [ ] Small Changes - Typos, formatting, slight revisions
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site and Docusaurus

**Purpose of the Pull Request:**
Adds new documentation for the Playmode Scenarios window and related properties.

**Issue Number:**
[DOCF-5362](https://jira.unity3d.com/browse/DOCF-5362)

### CODE REVIEW

File: mppm/troubleshoot/play-mode-scenario-troubleshoot.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+id: play-mode-scenario-troubleshoot
+title: Troubleshoot a test build
+description: Learn about errors that can happen when you build a play mode scenario
+---
+To troubleshoot an Error that appears when you enter Play Mode, refer to the Error message that appears in the console window. You can also open the [play mode scenario status dropdown](LINK) for more information.
```
Comment (jabbacakes) [inline mppm/troubleshoot/play-mode-scenario-troubleshoot.md:6]:
```suggestion
To troubleshoot an error that appears when you enter Play Mode, refer to the error message that appears in the console window. You can also open the [Play Mode scenario status dropdown](play-mode-scenario-dropdown-reference.md) for more information.
```

File: mppm/troubleshoot/play-mode-scenario-troubleshoot.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+To fix the error **Build profile needs to match the current platform**: 
+1. Open the Build Profile window ( **File** > **Build Profiles**). 
+2. In the **Platforms** list, select the platform that matches the platform that the Unity Editor is running on.
+
+## Additional resources
+* [Play Mode Scenarios requirements and limitations](play-mode-scenario-req.md)
+* [Play Mode scenario window reference](play-mode-scenario-window-reference.md)
+* [Unity Game Service (UGS) Game Server Hosting (Multiplay)](https://docs.unity.com/ugs/en-us/manual/game-server-hosting/manual/welcome-to-multiplay)
```
Comment (jabbacakes) [inline mppm/troubleshoot/play-mode-scenario-troubleshoot.md:15]:
```suggestion
* [Unity Gaming Services (UGS) Multiplay Hosting](https://docs.unity.com/ugs/en-us/manual/game-server-hosting/manual/welcome-to-multiplay)
```

**Review by jabbacakes — CHANGES_REQUESTED** [review]
Some inconsistencies here that need addressing, as well as that large and very confusing table on the Play Mode scenario window reference page.

File: mppm/mppm-reference/mppm-reference-lp.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+|**Property**|**Description**|
+|-|-|
+|[Instances status dropdown reference](play-mode-scenario-dropdown-reference.md)| Describes the meaning of each status icon in the Play Mode Scenarios dropdown.|
+|[Play Mode Scenarios window reference](play-mode-scenario-window-reference.md)| Describes the behavior of the properties in the Play Mode Scenarios window.|
+|[Play Mode status window reference](play-mode-status-window-reference)| Describes the behavior of the properties in the Play Mode status window. |
+|[Play Mode scenario instances status reference](play-mode-scenario-instances-status-reference.md)|Describes the meaning of each status icon in the instances status dropdown.|
+
+## Additional resources
+- [Multiplayer Roles reference](https://docs.unity3d.com/Packages/com.unity.dedicated-server@1.0/manual/multiplayer-roles-reference.html)
+- [Create a Play Mode Scenario](play-mode-scenario-create.md)
```
[the remark this answers is not in the document]
Comment (Vic-Cooper) [reply mppm/mppm-reference/mppm-reference-lp.md:16]: correct! I'll go through and manage it, Scenario should really only be capitalised when referring to the window name.

File: mppm/mppm-reference/play-mode-scenario-window-reference.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+||Multiplayer Role||The [multiplayer role](docs.unity3d.com/Packages/com.unity.dedicated-server@1.3/manual/multiplayer-roles.html) that this instance uses. Unity uses this Multiplayer Role and ignores the Multiplayer Role you assign in the Project Settings window. This property appears when the [Dedicated Server package](https://docs.unity3d.com/Packages/com.unity.dedicated-server@1.3/manual/index.html) exists in your project|
+||Tag||One or more reference words that you can use in a script to assign the main Editor Player to a certain behavior.
+To control whether a player exists on the client or server, use the Multiplayer Role property.|
+|Local Instances|||Instances of this project that run on your local machine.|
+||Name||The name of this instance that appears in the [Play Mode status window](play-mode-scenario-window-reference.md).|
+||Build Profile||The [Buld Profile](https://docs.unity3d.com/6000.0/Documentation/Manual/build-profiles.html) that this instance uses. When you build on a Windows device, select the Windows platform. When you build on a MacOS device, select the MacOS platform.|
```
[the remark this answers is not in the document]
Comment (Vic-Cooper) [reply mppm/mppm-reference/play-mode-scenario-window-reference.md:24]: oopth

File: mppm/mppm-reference/play-mode-scenario-window-reference.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+|||Stream Logs To Main Editor|Select the checkbox to display logs from this instance in the Unity Editor Console.|
+|||Logs Color|The color this instance uses to display logs in the Unity Editor.|
+|||Arguments|The [UGS launch parameters](https://docs.unity.com/ugs/manual/game-server-hosting/manual/concepts/launch-parameters) that modify this instance.|
+|Remote Instances|||Instances that exist in the [Unity Cloud](https://docs.unity.com/cloud/en-us) and use the [Unity Game Service (UGS) Game Server Hosting (Multiplay)](https://docs.unity.com/ugs/en-us/manual/game-server-hosting/manual/welcome-to-multiplay).|
+||Name||The name of this instance that appears in the [Play Mode status window](play-mode-scenario-window-reference.md).|
+||Multiplayer Role||The [multiplayer role](docs.unity3d.com/Packages/com.unity.dedicated-server@1.3/manual/multiplayer-roles.html) that this remote instance uses. Unity uses this Multiplayer Role and ignores the Multiplayer Role you assign in the Project Settings window. This property appears when the [Dedicated Server package](https://docs.unity3d.com/Packages/com.unity.dedicated-server@1.3/manual/index.html) exists in your project|
```
[the remark this answers is not in the document]
Comment (Vic-Cooper) [reply mppm/mppm-reference/play-mode-scenario-window-reference.md:32]: Same property, same description, applies to different instance.

File: mppm/mppm-reference/play-mode-scenario-window-reference.md [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+||Advanced Configuration|||Optional properties that you can use to debug and control this remote instance. |
+|||Fleet Name|The name of the server fleet to display in the Multiplayer Hosting environment. |
+|||Fleet Region|The server region this instance runs from.|
+|||Instance Amount of Cores|The amount of CPU cores this remote instance requests from the server.|
+|||Instance Memory|The amount of memory, in megabytes, that this instance requests from the server. To learn how much memory your project uses, use the [memory profiler package](https://docs.unity3d.com/Packages/com.unity.memoryprofiler@1.1/manual/index.html).|
+|||Instance CPU Clockspeed|The amount of CPU time, in megahertz, that this instance requests from the server.|
+|||Stream Logs To Main Editor|Select the checkbox to display logs from this instance in the Unity Editor Console.|
+|||Logs Color|The color of text that displays in the Unity Editor for logs that relate to this instance.|
+|||Arguments|The [UGS launch parameters](https://docs.unity.com/ugs/manual/game-server-hosting/manual/concepts/launch-parameters) that modify this instance.|
+
```
[the remark this answers is not in the document]
Comment (Vic-Cooper) [reply mppm/mppm-reference/play-mode-scenario-window-reference.md:42]: it is in the order in which the properties are presented. I do like the idea of beaking up the tables though. 
The content is grouped though, but I've just used multiple columns.

File: mppm/play-mode-scenario/play-mode-scenario-about.md [hunk 7]
```diff
@@ -0,0 +1,13 @@
+---
+id: play-mode-scenario-about
+title: Test live instances locally and remotely
+description: Learn how to use the Play Mode Scenarios window to test a multiplayer project
+---
+
+To test one or more instances of a multiplayer project on a local machine or a remote server, use the Play Mode Scenarios window. This window replicates the circumstances of a project that multiple users access from different environments that have different limitations.
```
[the remark this answers is not in the document]
Comment (Vic-Cooper) [reply mppm/play-mode-scenario/play-mode-scenario-about.md:7]: It's a landing page really, which means I need to add a ToC for it, thanks!

File: mppm/play-mode-scenario/play-mode-scenario-create.md [hunk 8]
```diff
@@ -0,0 +1,23 @@
+---
+id: play-mode-scenario-create
+title: Create a Play Mode Scenario
+description: Add a Play Mode Scenario to your project.
+---
+
+To create a new Play Mode scenario: 
+1. Install the Multiplayer Play Mode package (LINK).
+2. Select the Play Mode dropdown menu.
+3. Select **Configure play mode scenarios…**.
```
[the remark this answers is not in the document]
Comment (Vic-Cooper) [reply mppm/play-mode-scenario/play-mode-scenario-create.md:10]: yes, yes it is...

File: mppm/mppm-reference/play-mode-scenario-window-reference.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (jabbacakes) [reply mppm/mppm-reference/play-mode-scenario-window-reference.md:42]: Ah, fair enough. The table formatting is hard to parse in the raw markdown. But yeah, anything to make it clear that the things that seem the same, uh, aren't 😅

File: mppm/play-mode-scenario/play-mode-scenario-create.md (hunk 8 above)
[the remark this answers is not in the document]
Comment (jabbacakes) [reply mppm/play-mode-scenario/play-mode-scenario-create.md:10]: Oh 😢

**Review by johann-r-unity — APPROVED** [review]
I did not see anything shocking. Approved, thanks Vic! 
Side question ; do we have a staging website?  It's easier to read.

File: mppm/mppm-reference/mppm-reference-lp.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (Vic-Cooper) [reply mppm/mppm-reference/mppm-reference-lp.md:16]: Fixed all instances of this apitalisation in the next commit.

File: mppm/mppm-reference/play-mode-scenario-window-reference.md [hunk 9]
```diff
@@ -0,0 +1,46 @@
...
+description: Descriptions of the properties in the Play Mode Scenario window.
+---
+
+This table explains the behavior of each property in the Play Mode Scenarios window.
+
+|**Property**|||**Description**|
+|-|-|-|-|
+|Description|||Enter a description for this instance. This description exists as a tooltip for this Play Mode Scenario in the [Play Mode scenario dropdown](play-mode-scenario-dropdown-reference.md). |
+|Editor|||The instance that exists in the main Unity Editor.|
+||Multiplayer Role||The [multiplayer role](docs.unity3d.com/Packages/com.unity.dedicated-server@1.3/manual/multiplayer-roles.html) that this local instance uses. Unity uses this Multiplayer Role and ignores the Multiplayer Role you assign in the Project Settings window. This property appears when the [Dedicated Server package](https://docs.unity3d.com/Packages/com.unity.dedicated-server@1.3/manual/index.html) exists in your project.
```
[the remark this answers is not in the document]
Comment (jabbacakes) [reply mppm/mppm-reference/play-mode-scenario-window-reference.md:13]: @Vic-Cooper This one doesn't seem to be resolved

File: mppm/play-mode-scenario/play-mode-scenario-req.md [hunk 10]
```diff
@@ -0,0 +1,35 @@
...
+id: play-mode-scenario-req
+title: Play Mode Scenarios requirements and limitations
+description: Learn how to use the Play Mode Scenarios window to test a multiplayer project
+---
+
+To use the Play Mode Scenarios window, install the following packages: 
+* [Multiplayer Play Mode](https://docs-multiplayer.unity3d.com/tools/current/about/)
+* [Dedicated server](https://docs.unity3d.com/Packages/com.unity.dedicated-server@1.3/manual/index.html)
+
+A Play Mode Scenario that contains a local or remote instance requires a Build Profile that matches the platform your project exists on. To learn how to assign a Build profile, refer to [Create a Play Mode scenario](play-mode-scenario-create.md). Use a [Build Profile](https://docs.unity3d.com/6000.0/Documentation/Manual/build-profiles.html) to configure the type of build this scenario uses. To learn what each property in a build profile controls, refer to the [build profiles reference](https://docs.unity3d.com/6000.0/Documentation/Manual/build-profiles-reference.html).
```
[the remark this answers is not in the document]
Comment (jabbacakes) [reply mppm/play-mode-scenario/play-mode-scenario-req.md:11]: @Vic-Cooper Capitalisation question here still unresolved.

[the remark this answers is not in the document]
Comment (Vic-Cooper) [reply mppm/play-mode-scenario/play-mode-scenario-req.md:11]: hmnh. I'm gonna capitalize it consistently- the style guide has nothing for me here.

**Review by jabbacakes — APPROVED** [review]
👍
